src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/cluster.module.ts

```diff
@@ -81,7 +82,8 @@ import { UpgradeProgressComponent } from './upgrade/upgrade-progress/upgrade-pro
     NgbPopoverModule,
     NgbDropdownModule,
     NgxPipeFunctionModule,
-    NgbProgressbarModule
+    NgbProgressbarModule,
+    DashboardV3Module
```

Comment:
I don't like the idea of importing `DashboardV3Module` just for getting the health-check component. I see this is being used in more than one area so I think it should be more generalized. So can we move the cd-health-check component to `shared/components` module? That way it can be reused anywhere.